scaaml/capture/scope/ps6424e.py

```diff
@@ -532,7 +585,124 @@ def get_last_trace(self, as_int: bool = False) -> np.ndarray:
 
     def get_last_trigger_trace(self) -> np.ndarray:
         """Return a copy of the last trigger trace."""
-        return np.array(self._buffers[1][:])
+        # Digital trigger.
+        if self.trigger.is_digital:
+            bits = np.array(self._buffers[1][:], dtype=np.int16)
+            port_pin = self.trigger.port_pin
+            assert port_pin is not None
+            return np.array(bits & (2**port_pin), dtype=np.float32)
```

Comment:
Probably faster and more standard to replace the exponentiation by a bitshift here:
`2**port_pin` => `1 << port_pin`